recipe/activate-clang.sh

```diff
@@ -153,6 +153,14 @@ if [ "@CONDA_BUILD_CROSS_COMPILATION@" = "1" ]; then
   echo "cpu = '@UNAME_MACHINE@'" >> ${CONDA_PREFIX}/meson_cross_file.txt
   echo "cpu_family = '@MESON_CPU_FAMILY@'" >> ${CONDA_PREFIX}/meson_cross_file.txt
   echo "endian = 'little'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+  # specify path to correct binaries/metadata the meson will not auto-discover out of caution
+  # binaries from build env
```

Comment:
I'm not sure I understand this comment. Can you make it clear?

Reply:
So what I'm trying to say is that even though cmake/pkgconfig might be in build and/or host, meson will not auto-discover them and say "ah, this is the right one". It needs to be told explicitly via the cross-file.

Are we on the same page with that? If so, I can try to come up with a better formulation.

Reply:
I updated the comment & removed `pkg_config_libdir`. Could you PTAL?

Reply:
Gentle ping @isuruf